CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 <!--## [Unreleased]-->
 
+## [2.4.2] - 2020-05-22
```

Comment:
This would be a `v2.5.0` change since there are new features from the `Added` section.